lib/commands/general.js

```diff
@@ -28,8 +28,8 @@ commands.doSendKeys = async function (params) {
 commands.getDeviceTime = async function () {
   log.info('Attempting to capture android device date and time');
   try {
-    // format: Sat Jun 09 2018 11:17:42 GTM+0900 (JST)
-    let out = await this.adb.shell(['date', '\"+%a\ %b\ %d\ %Y\ %X\ GTM%z\ \(%Z\)\"']);
+    // format: 2018-06-09T16:21:54+0900
```

Comment:
it probably makes sense to mention this in the docstring, that the resulting date string format conforms to ISO-8601 by default.